charts/loki-distributed/values.yaml

```diff
@@ -103,6 +103,10 @@ loki:
     frontend:
       log_queries_longer_than: 5s
       compress_responses: true
+      
+    compactor:
+      working_directory: /var/loki/boltdb-shipper-compactor
```

Comment:
You should not specify that at all here. It is already configured as CLI flag. See https://github.com/unguiculus/loki-helm-chart/blob/master/charts/loki-distributed/templates/compactor/deployment-compactor.yaml#L47